добавил классы для шивровки и работы с файлами

Rarxan opened the pull request:
Добавлены основные 3 класса: Cipher, FileManager, MainApp.
Реализовал шифрование текста с записью в файл. Полёт нормальный =)

Rarxan committed:
добавил классы для шивровки и работы с файлами

Volodymyr95 commented on src/com/javarush/rybalchenko/cryptoanalyzer/Cipher.java:
```diff
@@ -0,0 +1,43 @@
+
+public class Cipher {
+
+    private static final char[] ALPHABET = {
```
show me in requirments where u shou use russian alphabet?

Volodymyr95 commented on src/com/javarush/rybalchenko/cryptoanalyzer/Cipher.java:
```diff
@@ -0,0 +1,43 @@
+        return -1;
+    }
+
+    public static String encrypt(String text, int key) {
```
why static?

Volodymyr95 commented on src/com/javarush/rybalchenko/cryptoanalyzer/MainApp.java:
```diff
@@ -0,0 +1,29 @@
+public class MainApp {
+    public static void main(String[] args) {
+
+//        String text = "Все, что можно вообразить – реально. – Пабло Пикассо.";
```
why do u push to review commented code?

Rarxan replied:
I tested this code, I will delete it

Volodymyr95 reviewed (commented):
prepare PR by requirments from doc

Rarxan committed:
Refactor: Improved project structure and refactor classes

Rarxan commented:
Updated the project with improvements:
- Added encryption and decryption functionality.
- Created classes for file handling.
- Fixed minor issues from previous review.

Volodymyr95 commented on src/com/javarush/rybalchenko/cryptoanalyzer/ConsoleReader.java:
```diff
@@ -0,0 +1,53 @@
+            String input = scanner.nextLine().toLowerCase();
+            try {
+                return OperationType.valueOf(input.toUpperCase());
+            } catch (IllegalArgumentException e) {
```
why do u catch uncked exception?

Volodymyr95 commented on src/com/javarush/rybalchenko/cryptoanalyzer/ConsoleReader.java:
```diff
@@ -0,0 +1,53 @@
+                return readKey();
+            }
+            return key;
+        } catch (NumberFormatException e) {
```
why do u catch uncheked exception?

Volodymyr95 commented on src/com/javarush/rybalchenko/cryptoanalyzer/OperationType.java:
```diff
@@ -0,0 +1,6 @@
+package com.javarush.rybalchenko.cryptoanalyzer;
+
+public enum OperationType {
```
bruteForce?

Volodymyr95 commented on src/com/javarush/rybalchenko/cryptoanalyzer/fileservice/FileManager.java:
```diff
@@ -0,0 +1,32 @@
+                text.append(line).append("\n");
+            }
+        } catch (FileNotFoundException e) {
+            System.out.println("File not found" + filepath);
```
why u only log in console? Create own exception and throw it

Rarxan committed:
Refactored ConsoleReader to use custom ValidationException

Rarxan committed:
Added brute force decryption and finished the Caesar Cipher project

Rarxan committed:
Implemented Caesar Cipher decryption, encryption, and brute force attack functionality. All features are working correctly.
